src/config/constants.ts

```diff
@@ -306,6 +308,7 @@ export const DOCUMENTATION = {
     DEPLOYMENT: `${DOCUMENTATION_HOME_PAGE}/v/v0.6/usage/applications/creating-application/deployment-template/deployment`,
     WEBHOOK_API_TOKEN: `${DOCUMENTATION_HOME_PAGE}/v/v0.6/getting-started/global-configurations/authorization/api-tokens`,
     WEBHOOK_CI: `${DOCUMENTATION_HOME_PAGE}/v/v0.6/usage/applications/creating-application/ci-pipeline#3.-deploy-image-from-external-service`,
+    APP_TAGS: `${DOCUMENTATION_HOME_PAGE}/v/v0.6/`
```

Comment:
Seems like an incomplete URL. Can you please confirm?

Reply:
Will do before merging. it's not ready yet